File: src/test/resources/same/literals/Literals.go
package main

import "fmt"

func main() {
    param1 := 1
    param2 := "stringParam2"
    param3 := 0xFFFFFFFFFFFF
    var param4 uint64 = 0xff1a618b7f65ea12
    var param5 uint64 = 0xc4ceb9fe1a85ec53
    param6 := 0757
    param7 := 0123546263753256452432
    param8 := 123.E+2
    param9 := 1e-1
    param10 := -1e-1
    param11 := 1.e+0i

    fmt.Println(param1,param2,param3,param4,param5)
    fmt.Println(param6,param7,param8,param9,param10)
    fmt.Println(param11)
}

File: src/test/resources/same/literals/Literals.js
var param1 = 1;
const param2 = 2;
let param3 = 3;
let param4 = "stringParam4";

// Strict mode in ECMAScript 5 forbids octal syntax.
// Octal syntax isn't part of ECMAScript 5, but it's
// supported in all browsers by prefixing the octal
// number with a zero.
let param5 = 0777;
let param6 = 0888;

// ES6+
let param7 = 0o10;
let param8 = -0o10;

let param9 = 0b01111111100000000000000000000000;
let param10 = 0B00000000011111111111111111111111;
let param11 = 0x123456789ABCDEF;
let param12 = 0xFFFFFFFFFFFFFFFF;
let param13 = -1.2e55;
let param14 = 1.2e-55;
let param15 = -1.2e-55;
let param16 = .1E3;